examples/losses_triplet.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>nit: the new tfa loss</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Good catch thanks! I've removed the line</p>